src/main/play-doc/api/BundleAPI.md

```diff
@@ -41,7 +41,7 @@ Cache-Control: max-age={max-age}
 
 Field        | Description
 -------------|------------
-location-url | The location of the requested service including any trailing parts to the path requested e.g. `/customers/123` would result in `http://10.0.1.22:8080/customers/123` supposing that ConductR's proxy service's address is `10.0.1.22`, and the endpoint service port is `8080`. If the protocol for the service was TCP then that will be reflected in the returned location's protocol field. For example looking up `/jms` may yield `tcp://10.0.1.22:61616` supposing that the Active/MQ JMS broker's service is offered on port 61616.
+location-url | The location of the requested service including any trailing parts to the path requested e.g. `/customers/123` would result in `http://10.0.1.22:10121/customers/123` supposing that service's host address is `10.0.1.22`, and the endpoint service host port is `10121`. If the protocol for the service was TCP then that will be reflected in the returned location's protocol field. For example looking up `/jms` may yield `tcp://10.0.1.22:10121`.
```

Comment:
@huntc - I know this statement has been around in the existing doc:

> For example looking up `/jms` may yield `tcp://10.0.1.22:10121`

Is this statement correct? If I were to lookup `/jms` through the service locator, will it return HTTP `307` with `tcp://10.0.1.22:10121` in the `Location` field?